src/libs/ReportActionsUtils.js

```diff
@@ -170,6 +170,25 @@ function getMostRecentIOURequestActionID(reportActions) {
     return _.last(sortedReportActions).reportActionID;
 }
 
+/**
+ * Returns array of links inside a given report action
+ *
+ * @param {Object} reportAction
+ * @returns {Boolean}
+ */
+function extractLinksFromMessageHtml(reportAction) {
+    const htmlContent = lodashGet(reportAction, ['message', 0, 'html']);
+
+    // Regex to get link in href prop inside of <a/> component
+    const regex = /<a\s+(?:[^>]*?\s+)?href="([^"]*)"/gi;
+
+    if (!htmlContent) {
+        return;
```

Comment:
BTW this caused a bug which I fixed here https://github.com/Expensify/App/pull/21369/files

Reply:
Thanks for catching it 🙇